style.css

```diff
@@ -0,0 +1,264 @@
+*, *::after, *::before {
+    box-sizing: border-box;
+} 
+
+body {
+    font-size: 10px;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    display: flex;
+    flex-direction: column;
+    background-color: #000000;
+}
+
+header {
+
+    background-color: #000000;
+    background-image: url(images/northernLights.jpg);
+    background-size: 90rem 30rem;
+    background-repeat: no-repeat;
+    background-position: center;
+    min-height: 10rem;
+    align-items: baseline;
+    /* padding: 2rem 1rem; */
+}
+
+a {
+    text-decoration: none;
+}
+
+h1 {
+    font-size: 4rem;
+    text-align: center;
+    font-weight: 600;
+}
+
+.navLinks {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.navLinks a {
+    line-height: 2rem;
+    color: #000000;
+    text-decoration: none;
+    padding: 2rem;
+}
+
+.jumbotron {
+    font-size: 1.5rem;
+    /* background-image: url(images/redcircuit.jpg);
+    background-size: 90rem 70rem;
+    background-repeat: no-repeat;
+    background-position: center; */
+    /* I have removed picture temporarily as it clashes */
+    /* background-color: #0e5b1a; */
+    background-color: #000000;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 4rem;
+    min-height: 15rem;
+}
+
+.decorativeBox img {
+    min-width: 15rem;
+    min-height: 15rem;
+    max-height: 30rem;
+    max-width: 20rem;
+    border-radius: 200px;
+}
+
+.jumbotronTextbox {
+    min-width: 15rem;
+    min-height: 15rem;
+    max-height: 40rem;
+    max-width: 30rem;
+    margin-left: 4rem;
+}
+
+h2 {
+    color: #ffffff;
+    background-color: #000000;
+}
+
+.jumbotron p {
+    display: inline;
```

Comment:
Up to you but I would remove this as isnt the nicest way to achieve the effect its giving. also this style isnt consistent with other blocks of text on the page so feels a little out of place UX wise. your choice tho!

Reply:
Yeah i think i will leave it for now as i quite like it as a statement piece. I will look into another way of achieving this styling